examples/hnas/abohb_example.py

```diff
@@ -0,0 +1,229 @@
+import autogluon as ag
```

Comment:
Very cool. Let us also replace the other examples/tutorials/etc (MXNet code) by PyTorch.